src/EnergyPlus/SingleDuct.cc

```diff
@@ -4887,7 +4952,42 @@ namespace SingleDuct {
 		if ( BeginDayFlag ) {
 		}
 
-		if ( FirstHVACIteration ) {
+		if ( MyEnvrnFlag( ATMixerNum ) ) {
+			if ( ZoneEquipConfig( SysATMixer( ATMixerNum ).ZoneEqNum ).AirLoopNum > 0 ) {
+				SysATMixer( ATMixerNum ).AirLoopNum = ZoneEquipConfig( SysATMixer( ATMixerNum ).ZoneEqNum ).AirLoopNum;
+				MyEnvrnFlag( ATMixerNum ) = false;
+			}
+		}
+
+		Real64 mDotFromOARequirement( 0.0 );
+		Real64 vDotOAReq( 0.0 );
+		if ( !SysATMixer( ATMixerNum ).NoOAFlowInputFromUser ) {
+			Real64 airLoopOAFrac( 0.0 );
+			if ( SysATMixer( ATMixerNum ).AirLoopNum > 0 ) {
+				airLoopOAFrac = DataAirLoop::AirLoopFlow( SysATMixer( ATMixerNum ).AirLoopNum ).OAFrac;
+				if ( airLoopOAFrac > 0.0 ) {
+					if ( SysATMixer( ATMixerNum ).OARequirementsPtr > 0 ) {
+						vDotOAReq = CalcDesignSpecificationOutdoorAir( SysATMixer( ATMixerNum ).OARequirementsPtr, SysATMixer( ATMixerNum ).ZoneNum, true, true );
+					} else {
+						vDotOAReq = CalcDesignSpecificationOutdoorAir( ZoneSizingInput( SysATMixer( ATMixerNum ).SizingInputNum ).ZoneDesignSpecOAIndex, SysATMixer( ATMixerNum ).ZoneNum, true, true );
```

Comment:
This will fail if SizingInputNum is zero.  If OARequirementPtr is set earlier (as suggested) then this if block isn't necessary.  Just keep line 4970.